development/build/index.js

```diff
@@ -60,7 +60,8 @@ function defineAndRunBuildTasks() {
     skipStats,
   } = parseArgv();
 
-  const browserPlatforms = ['firefox', 'chrome', 'brave', 'opera'];
+  // const browserPlatforms = ['firefox', 'chrome', 'brave', 'opera'];
+  const browserPlatforms = ['chrome'];
```

Comment:
do we want this as part of our pull request to the original metamask repo?
from what I understand this is just to reduce the work of the hot-reload right?